Fly_Swatter/mode_control.py

```diff
@@ -20,9 +20,12 @@ def system_run(search_runs = 1, seed_search = None, seed_fire = None, verbose =
       graphical: default = false, provides for if a graph should be printed or not
       realism: default = 0, 0 provides for laser projectile, gravityless, no air resistance; 1 provides for gravity based projectile
   """
-  target_mode = search.search_mode(runs = search_runs, seed = seed_search)
-  if target_mode:
-    log = fire_mode.track_lock(seed = seed_fire, realism = realism)
+  fire_on, first_loc = search.search_mode(runs = search_runs, seed_fire = seed_fire, seed_search = seed_search)
```

Comment:
going to want to switch out the fire boolean for a argument in system run for firing